report/MVP/MVP.tex

```diff
@@ -0,0 +1,23 @@
+This chapter will talk about what our Minimum Viable Product (MVP) is, and we will in the following chapters go though how we have developed this MVP.
+
+\section{What is our MVP?}
+An MVP is a piece of product, that gives some sort of value, with the least amount of work went into it.
+It will not have many advanced features, but just showcase the basic functionalities, so the product can be tested, and further be worked on, to get all of the wanted features.
```

Comment:
```suggestion
It will not have many advanced features, but will just showcase the basic functionalities of the product, so the product can be tested, and further futher developemnt can be based upon the test, to get all of the wanted features.
```

Reply:
```suggestion
It will not have many advanced features, but will just showcase the basic functionalities of the product, so the product can be tested, and futher developemnt can be based upon the test, to get all of the wanted features.
```